backend/python/app/services/implementations/story_service.py

```diff
@@ -125,23 +125,28 @@ def get_story_translations(self, user_id, translator, language, level):
 
     def get_story_translation(self, id):
         try:
-            return (
+            translation = StoryTranslation.query.get(id)
+
+            story_details = (
                 db.session.query(
                     Story.id.label("story_id"),
                     Story.title.label("title"),
                     Story.description.label("description"),
                     Story.youtube_link.label("youtube_link"),
                     Story.level.label("level"),
-                    StoryTranslation.id.label("story_translation_id"),
```

Comment:
Should this be called `id` or `story_translation_id`? For consistency, it looks like id, but it seems less clear than the alternative

Reply:
now that i think about it, lets do `id`, consistency will help frontend development